src/ray/core_worker/actor_manager.cc

```diff
@@ -129,31 +121,33 @@ bool ActorManager::AddNewActorHandle(std::unique_ptr<ActorHandle> actor_handle,
   const auto actor_creation_return_id = ObjectID::ForActorHandle(actor_id);
   // Detached actor doesn't need ref counting.
   if (!is_detached) {
-    // We don't need to add an initial local ref here because it will get added
-    // in AddActorHandle.
     reference_counter_->AddOwnedObject(actor_creation_return_id,
                                        /*inner_ids=*/{},
                                        caller_address,
                                        call_site,
                                        /*object_size*/ -1,
                                        /*is_reconstructable=*/true,
-                                       /*add_local_ref=*/false);
```

Comment:
Is `/*add_local_ref=*/false` a bug?

Reply:
It used to be right because we would add the local ref in the call to AddActorHandle, but now we might as well do it when we add the owned object.